sqlglot/tokens.py

```diff
@@ -517,6 +518,9 @@ def _quotes_to_format(
                 semicolon=_TOKEN_TYPE_TO_INDEX[TokenType.SEMICOLON],
                 string=_TOKEN_TYPE_TO_INDEX[TokenType.STRING],
                 var=_TOKEN_TYPE_TO_INDEX[TokenType.VAR],
+                heredoc_string_alternative=_TOKEN_TYPE_TO_INDEX[
+                    klass.HEREDOC_STRING_ALTERNATIVE or TokenType.VAR
```

Comment:
```
or TokenType.VAR
```
I don't see the same logic being applied to the Python tokenizer. Is this expected?

Reply:
Mirroring would be easier to maintain for sure. Less cognitive load

Reply:
agree rust should try to mimic python implementation whenever possible